cdap-docs/admin-manual/source/index.rst

```diff
@@ -69,10 +69,10 @@ introduces the CDAP Console.**
 
 - |metrics|_ CDAP collects **metrics about the application’s behavior and performance**.
   
-.. |runtime-arguments| replace:: **Runtime Arguments:**
-.. _runtime-arguments: operations/runtime-arguments.html
+.. |preferences| replace:: **Preferences:**
+.. _preferences: operations/preferences.html
```

Comment:
When you change a filename (or add or delete a file), you need to find the index.rst or table-of-contents.rst that refers to it, and change that accordingly.
In this case, it's the ``operations/index.rst``, in the table of contents near the top of the file:

```
.. toctree::
   :maxdepth: 1
   
    Logging <logging>
    Metrics <metrics>
    Runtime Arguments <runtime-arguments>
    Scaling Instances <scaling-instances>
    CDAP Console <cdap-console>
    Troubleshooting <troubleshooting>
```